### ISSUE TITLE
[iptorrents] Your cookie did not work: Your cookie did not work (Config)

### ISSUE THREAD

**Issue by deegan** [Jackett/Jackett#6248]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue. Duplicating issues makes it more difficult for everyone to follow. Your issue may have already been solved in the past as well.

Provide a description of the feature request or bug, the more details the better.

If you are experiencing an issue with a tracker, then:
1. Use your browser to check you can access the site directly, and if a login is required, check you can login and that you do not have any outstanding account issues.
2. If you haven't already, try upgrading to the latest version of Jackett, your issue may have already been resolved.
3. If it is still not working for you, then a **full enhanced log must be included**. Instructions for obtaining logs are here: https://github.com/Jackett/Jackett#troubleshooting

(At the bottom of your Jackett Dashboard page) **Jackett version**:  
(if you installed using Jackett.Binaries.Mono.tar.gz) **Mono version**:

-------------------------------
Not exactly sure when this tracker stopped working. I do nightly restarts of the Docker container that Jackett is running in to get the latest version. I tried updating the cookie but just get an error. I have a couple other trackers that both work in Jackett.

----- Jackett version ------
 Jackett Version 0.12.863.0 
----- Jackett version ------

----- Jackett log start -----
System.Exception: Your cookie did not work: Your cookie did not work
   at Jackett.Common.Indexers.IPTorrents.ApplyConfiguration(JToken configJson) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/IPTorrents.cs:line 197
   at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[] config) in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/IndexerApiController.cs:line 94
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/appveyor/projects/jackett/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 73
----- Jackett log end -----

**Comment by garfield69** [comment]
can't test for myself because my ip block is not required to be challenged by a cloudflare captcha, so my iptorrent jackett indexer just needs the username and password to login.

are you certain you pasted the complete cookie string that you got from a browser running on the same computer that is running jackett?
my browser shows me that the cookie string looks like `__cfduid=d**redacted**6; uid=1redacted0; pass=c**redacted**2; hideCats=0; hideTop=0`
does your include all of those cookie keywords (the last two are probably not significant).?

**Comment by deegan** [comment]
You know what you are right. When I opened the settings for iptorrents it wanted the cookie, so I filled it in but got an error. Now after last night the container was restarted and I refreshed the page, Iptorrents settings now want me to use username and password and that works just fine.

So yea strange it did not work with the cookie, it only showed me to enter the __cfduid value though in the example so that could have been it. But let's close this ticket because the method for signing in had changed and my view did not reflect that change. It's all working fine now.

Thanks for quick response. <3
